api/views/course.py

```diff
@@ -184,20 +184,33 @@ def course_grade_book(self, request, pk):
         for event in events.all():
             teams.extend(event.team_set.all())
 
+        from course.models.models import UserQuestionJunction, Submission
         results = []
         for team in teams:
             for course_reg in team.course_registrations.filter(registration_type="STUDENT", status="VERIFIED"):
+                attempts = []
                 consent = course_reg.user.consents.last()
-                results.append(
-                    {
-                        "grade": team.tokens_received,
-                        "total": team.event.total_tokens,
-                        "name": course_reg.name,
-                        "event_name": team.event.name,
-                        "legal_first_name": consent.legal_first_name if consent else "",
-                        "legal_last_name": consent.legal_last_name if consent else "",
-                        "student_number": consent.student_number if consent else "",
-                    }
-                )
+                question_ids = [question.id for question in team.event.question_set.all().order_by('title')]
+                for question_id in question_ids:
+                    uqj = UserQuestionJunction.objects.get(id=question_id)
```

Comment:
```suggestion
                    uqjs = question.question_junctions
```

Reply:
or it might be something similar to that I'm not sure about the exact term

Reply:
The closest I could find was question.user_junctions which as far as I can tell only have the fields: 
id, user_id, question_id, random_seed, last_viewed, opened_tutorial, tokens_received, solved_at, is_solved, is_partially_solved, is_favorite
I can't find something that will give me access to the submissions. Is there a particular place I should be looking that I haven't yet?